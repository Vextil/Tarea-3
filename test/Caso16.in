# stress ColaPrioridad
# Prueba de orden de tiempo
TiempoColaPrioridad
# Prueba de uso de memoria
MemoriaColaPrioridad
Fin
