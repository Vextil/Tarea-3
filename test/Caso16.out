1># stress ColaPrioridad
2># Prueba de orden de tiempo
3>Fin EsVacia, Minimo, Prioridad. Fin Extraer. Fin Elementos. Fin Insertar. 
4># Prueba de uso de memoria
5>Fin Destruir. Fin Extraer. 
6>Fin
